src/components/navbar-header-panels/dropdown-menu-panel.component.tsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import { render } from "react-dom";
+import "carbon-components/css/carbon-components.min.css";
+import { Dropdown } from "carbon-components-react";
+
+const items = [
```

Comment:
What are these?

Reply:
I initially wanted to add the dropdown as a react module without carbon design

Reply:
I just forgot to clean up the folder before pushing